packages/aws-cdk/lib/init-templates/app/dotnet/src/HelloCdk/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Amazon.CDK;
+using System;
+
+namespace HelloCdk
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var app = new App(args);
+
+            new HelloStack(app, "hello-cdk-1", new StackProps());
```

Comment:
Why do you need to specify StackProps if it's empty?

Reply:
Maybe the generated code doesn't allow default parameters?

Reply:
This is due to my oversight in the generator--it doesn't generate default values for arguments with optional types. I thought there was an issue for this, but I don't see one. New issue is here: https://github.com/awslabs/jsii/issues/210.

In the meantime, we can pass `null` rather than `new StackProps()`.